mani_skill2/vector/wrappers/sb3.py

```diff
@@ -23,24 +23,27 @@ def select_index_from_dict(data: dict, i: int):
 
 
 class SB3VecEnvWrapper(SB3VecEnv):
-    """A wrapper for SB3 VecEnv to make it compatible with ManiSkill2 VecEnv."""
+    """A wrapper for ManiSkill2 VecEnv to make it compatible with SB3 VecEnv"""
 
     def __init__(self, venv: VecEnv):
-        super().__init__(venv.num_envs, venv.observation_space, venv.action_space)
+        super().__init__(venv.num_envs, venv.single_observation_space, venv.single_action_space)
         self.venv = venv
+        self._last_seed = None
 
     def seed(self, seed: Optional[int] = None) -> List[Union[None, int]]:
```

Comment:
Needed as sb3 uses seed still, despite switching to gymnasium. There is actually small bug in this wrapper that was just fixed but otherwise works as intended.